eeraster.py

```diff
@@ -281,5 +281,26 @@ def dem():
       'projects/sat-io/open-datasets/GLO-30').select("b1"))
   return _dem
 
+def ordinary_kriging_means():
+  """
+  Returns an eeRaster representing the mean oxygen isotope value with an isoscape
+  generated using Ordinary Kriging.
+  """
+  eeddf.initialize_ddf()
+  global _ordinary_kriging_means
```

Comment:
This is declared as a global but appears to be missing from the globals at the top of the file.